### DISCUSSION TITLE
Sightseeing Log

### DISCUSSION THREAD

**Discussion by tacitfirefox** [mattantonelli/ffxiv-collect#13] [Ideas]
Hello!
I've been following and using FFXIV-Collect for some time now (I LOVE it!). A newer feature I've noticed in the Tools section, which I don't see via your Postman documentation. As I am now playing through from a fresh character, the tool I find I end up using every time to complete my Sightseeing Log is https://tylian.net/sslog/. I propose it's inclusion under the Tool submenu, ideally on a character-specific basis. Is this something that sounds doable or interesting? I know I would find value in it.

**Comment by mattantonelli** [comment]
I've had this requested before, but unforutunately it's at the bottom of my list since it's already well implemented on other sites like the one you provided. In particular, having to write a script to implement the in-game clock and weather system just for this one feature is what makes me loathe to sit down with it.
